udata/templates/macros/certified.html

```diff
@@ -1,9 +1,8 @@
 {% macro badge_if_certified(org) %}
 {% if org.public_service %}
-<img src="{{ theme_static('img/certified-stamp.png') }}" alt="certified"
-    class="certified" data-toggle="popover"
-    data-title="{{ _('Certified public service') }}"
-    data-content="{{_('The identity of this public service is certified by %(certifier)s', certifier=config.SITE_AUTHOR)}}"
-    data-container="body" data-trigger="hover"/>
+<img src="{{ theme_static('img/certified-stamp.png') }}" alt="certified" class="certified"
```

Comment:
Yes, good idea, but in another PR I think.
I just switched from the bootstrap popover to the popover directive.
We need to see if a component is the right way to go because right now it works without javascript (for the stamp, not the popover part)